File: moreapps/src/main/java/io/github/raghavsatyadev/moreapps/listener/MoreAppsDownloadListener.java
package io.github.raghavsatyadev.moreapps.listener;

import androidx.annotation.NonNull;

import io.github.raghavsatyadev.moreapps.MoreAppsDialog;
import io.github.raghavsatyadev.moreapps.model.MoreAppsDetails;

import java.util.List;

public interface MoreAppsDownloadListener {
    void onSuccess(MoreAppsDialog moreAppsDialog, @NonNull List<MoreAppsDetails> moreAppsDetails);

    void onFailure();
}

File: moreapps/src/main/java/io/github/raghavsatyadev/moreapps/listener/MoreAppsDownloadListener.kt
package io.github.raghavsatyadev.moreapps.listener

import io.github.raghavsatyadev.moreapps.MoreAppsDialog
import io.github.raghavsatyadev.moreapps.model.MoreAppsDetails

interface MoreAppsDownloadListener {
    fun onSuccess(moreAppsDialog: MoreAppsDialog, moreAppsDetails: List<MoreAppsDetails>)
    fun onFailure()
}